src/lib/CopyBSR.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
+  import Fa from 'svelte-fa'
+  import { faTwitch } from '@fortawesome/free-brands-svg-icons'
+  import { onMount } from 'svelte'
+  import { fade } from 'svelte/transition'
+
+  export let fontSize: string = '1rem'
+  export let mapId: string
+
+  // Adding props to allow these to be customizable for the MOTW Section
+  export let tooltipTop: string = '32px'
```

Comment:
These properties are annoying to maintain manually and it's prone to breaking. I have opened a PR to your branch with an alternative solution. Use it as you like.